src/react-native/07-debugging-devtools/debugging-devtools.md

```diff
@@ -10,39 +10,78 @@
 
 In this section, you will:
 
-- TODO
+- [resource](https://reactnative.dev/docs/debugging)
 
-## Objective 1: TODO
+## Objective: Devtools setup
```

Comment:
Will this end up being two objectives, one for React DevTools and another for the experimental debugger?

Reply:
Yes, I just thought about this last night.